pkg/engine/imageVerify.go

```diff
@@ -385,36 +394,27 @@ func (iv *imageVerifier) verifyAttestations(
 		var attestationError error
 		path := fmt.Sprintf(".attestations[%d]", i)
 
-		attestors := attestation.Attestors
 		if len(attestation.Attestors) == 0 {
-			attestors = []kyvernov1.AttestorSet{{}}
+			return ruleResponse(*iv.rule, response.ImageVerify, path+": missing attestors", response.RuleStatusFail), ""
```

Comment:
This makes `attestor` required in `attestations`, if so, we need to change the policy validation logic.